include/vicon_transformer/o80_standalone.hpp

```diff
@@ -0,0 +1,65 @@
+/**
+ * @file
+ * @brief o80 Standalone implementation for Vicon system.
+ * @copyright 2023 Max Planck Gesellschaft.  All rights reserved.
+ */
+#pragma once
+
+#include <o80/standalone.hpp>
+#include <o80/void_state.hpp>
+
+#include "o80_driver.hpp"
+#include "types.hpp"
+
+namespace vicon_transformer
+{
+constexpr int STANDALONE_QUEUE_SIZE = 50000;  // TODO: what is a good value?
```

Comment:
I simply copied the value from tennicam_client.  From looking at the documentation it seems that this is only relevant for commands sent to the driver.  Is that correct?  Since the Vicon system does not accept commands, could I just set it to 0 (or maybe 1) to save memory?

Reply:
It is also used for the size of the observations history
(as a side note: making this number smaller also allows for applications to start faster)

Reply:
I see.  Is there some rule of thumb how much history we should keep?  Maybe also a question for @dtrb.
The Vicon system runs at 300 Hz, so the current queue size would cover 166 seconds.  Probably less would be enough?

Reply:
2 / 3 minutes of history seem good to me

Reply:
Ok, then let's keep the current value.